core/cas-server-core-logout/src/main/java/org/apereo/cas/logout/DefaultSingleLogoutServiceLogoutUrlBuilder.java

```diff
@@ -18,6 +19,9 @@
 public class DefaultSingleLogoutServiceLogoutUrlBuilder implements SingleLogoutServiceLogoutUrlBuilder {
     private static final Logger LOGGER = LoggerFactory.getLogger(DefaultSingleLogoutServiceLogoutUrlBuilder.class);
 
+    @Autowired
+    private UrlValidator urlValidator; 
```

Comment:
Please don't autowire. Pass the instance directly when the builder is constructed in its own bean.